src/internal/views/nunjucks/view-licences/rename.njk

```diff
@@ -9,7 +9,7 @@
 
   {{ title('Licence number ' + summary.licenceNumber, summary.documentName ) }}
 
-  <div class="well">
+  <div class="inset-text--rename-licence">
```

Comment:
Hopefully you should find the `rename` feature has been removed from the internal user site, so it should be OK to delete this template.

Reply:
File now deleted